charts/pega/README.md

```diff
@@ -480,6 +480,17 @@ Parameter           | Description    | Default value
 `hpa.enableCpuTarget` | Set to true if you want to enable scaling based on CPU utilization or false if you want to disable it | true
 `hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it (Pega recommends leaving this disabled) | false
 
+### Setting a Pod Disruption Budget (PDB)
+You can configure a Kubernetes `PodDisruptionBudget` on your tier by setting `pdb.enabled` to `true`.  For more details, see the Kubernetes 
+[Pod Disruption Budgets documentation](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets).
+
+The following parameters are mutually exclusive.  You must specify either `pdb.minAvailable` or `pdb.maxUnavailable` but not both.
```

Comment:
I think you should state that clients configure this in their deployment values.yaml file and it's disabled ('false') by default. Also, mention that you must use comments to disable the other setting.

Finally, the values.yaml files each have a 50% value commented out, not an integer so I'm automatically confused.